.github/workflows/ci.yml

```diff
@@ -709,3 +709,22 @@ jobs:
         run: cd ${{ matrix.crate }} && cargo test --release loom
         env:
           RUSTFLAGS: --cfg loom -Cdebug-assertions
+
+  xdp-bpf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - uses: camshaft/install@v1
+        with:
+          crate: bpf-linker
+
+      - uses: camshaft/rust-cache@v1
+
+      - name: Build ebpf
```

Comment:
I guess we only test that our build is identical and that we can load the program, but we do not yet do an e2e test that the XDP BPF module correctly routes the packets? 

Will we do so subsequently by sending in malformed packets and show they are not received, while the valid ones are?

Reply:
The XDP IO provider will be coming soon. This PR is just focused on the default BPF program that ships with said IO provider.